src/components/ActionDataSamplesCard.tsx

```diff
@@ -155,18 +165,21 @@ const DataSamplesRowCard = ({
   );
 };
 
+interface RecordingAreaProps extends BoxProps {
```

Comment:
Do you mean `StackProps`?
```suggestion
interface RecordingAreaProps extends StackProps {
```

Reply:
I've not tended to do this when the client code doesn't need to / shouldn't be relying on a particular root-level component.